src/ui/buttons/buttonDesigns/fadeOutLogoShowLabelText.jsx

```diff
@@ -0,0 +1,121 @@
+/* @flow */
+/** @jsx node */
+import { LOGO_CLASS } from '@paypal/sdk-logos/src';
+import { node, Fragment, type ChildType } from 'jsx-pragmatic/src';
+
+import { CLASS } from '../../../constants';
+import { BUTTON_SIZE_STYLE } from '../config';
+
+import type {  LabelOptions, DivideLogoAnimationProps } from './types';
+
+export const FADEOUT_LOGO_SHOW_LABEL_TEXT_CONFIG = {
+    tiny:       { min: BUTTON_SIZE_STYLE.tiny.minWidth },
+    medium:     { max: BUTTON_SIZE_STYLE.medium.maxWidth },
+    cssClasses: {
+        DOM_READY:                  CLASS.DOM_READY,
+        ANIMATION_CONTAINER:        'personalized-design-container',
+        PAYPAL_LOGO:                LOGO_CLASS.LOGO,
+        ANIMATION_LABEL_CONTAINER:  'personalized-label-container',
+        PAYPAL_BUTTON_LABEL:        CLASS.BUTTON_LABEL
+    }
+};
+
+export function FadeoutLogoShowLabelTextComponent({ designLabelText } : LabelOptions) : ChildType {
+    const CONTAINER_CLASS = FADEOUT_LOGO_SHOW_LABEL_TEXT_CONFIG.cssClasses.ANIMATION_CONTAINER;
+    const LABEL_CLASS = FADEOUT_LOGO_SHOW_LABEL_TEXT_CONFIG.cssClasses.ANIMATION_LABEL_CONTAINER;
```

Comment:
This is the same for all designs so you can remove it and use the hardcoded class where necessary.

Reply:
I prefer to use this variable, since I am using this  class in many places. So whenever we need to change the class, I just need to come to here and change the value, instead of changing it every where.